admin_manual/configuration_ldap/ldap_proxy_cache_server_setup.rst

```diff
@@ -75,36 +78,138 @@ With that done, restart OpenLDAP by running the following command::
+
+
+5. Perform a test search
 ------------------------
 
 Now that the server's installed, configured, and running, we next need to perform a search. 
 This will check that records are being correctly cached.
 To do so, update the command below with values from your Active Directory server configuration, and then run it.::
 
-  ldapsearch -H ldap://localhost -x -b "cn=users,dc=ad,dc=YOUR_DOMAIN,dc=com" -D "cn=users,dc=ad,dc=YOUR_DOMAIN,dc=com"  -LLL "(sn=Name)" -w "Password"
-
-.. note::
-   If you see warnings in the console output, they are not crucial.
+  ldapsearch -h localhost -x -LLL -D "cn=admin,cn=users,dc=example,dc=com" -b "cn=users,dc=example,dc=com" -w "Password" "(cn=Administrator)" name
 
-5. Check the Logs 
------------------
+or
 
-If the query was cached, then it will be visible in the logs. 
-To check, run the following command.::
+  ldapsearch -H ldaps://localhost:636 -x -LLL -D "cn=admin,cn=users,dc=example,dc=com" -b "cn=users,dc=example,dc=com" -W "(cn=Administrator)" name
 
-  tail -f /var/log/syslog | grep QUERY
+-h = host adress (Example: localhost or 192.168.1.1)
+-H = host adress (Example: ldap:// or ldaps:// hostname or ip and port :389 or :636 )
+-x = simple authentication
+-b = Search Base, (Example: "cn=Users,dc=example,dc=com")
+-D = User with permissions (Example: "cn=Admin,dc=example,dc=com")
+-LLL = Show only results, no extra information
+-w = Password ("Password")
+-W = Password, will ask for password and hide your input
+"(cn=Administrator)" = Filter the search
+name = Show only this attributes
 
 If you see results including: ``"Query cachable"`` and ``"Query answered (x) times"``, then the setup works. 
 
+
+6. Configure ownCloud LDAP App
+--------------------------------
+
+Login as ownCloud admin in your ownCloud server.
+
+Click on the dropdown menu in the top-left corner next to "Files", then on Apps. 
+
+Click on "Not enabled", and enable the "LDAP user and group backend" App.
+
+Go to the admin dropdown menu in the top-right corner, select "Admin".
+
+In the administration section, click on the left side on "LDAP".
+
+
+Configure Server-Tab
+
+First enter the server address, either IP or DN.
+
+You can click on the button to detect your servers port or enter it manualy.
+
+Next, enter the dn of the user, you want to log in with and in the next line enter the password.
+
+Then you can click on "detect base dn" or enter it manualy. then click on "test base dn".
+
+If you fullfill all requirement you should get a green light and "configuration ok" message.
+
+
+Configure Users
+
+Select the objectclass for the users, for example "user".
+
+Verify your settings, you will see the number of users being found.
+
+
+Configure Login Attributes
+
+A configuration appears by default, adjusted to the users configuration, 
+if required, adjust the login paramethers additional login attributes.
+
+
+You can check users with any of the allowed login options. You can adjust them or let them the way they are.
```

Comment:
In this sentence, change "_let them the way they are_" to "_leave them the way they are_"